stubs/pyinstaller/PyInstaller/utils/hooks/__init__.pyi

```diff
@@ -0,0 +1,79 @@
+# https://pyinstaller.org/en/stable/hooks.html
+
+import os
+from _typeshed import AnyOrLiteralStr, StrOrBytesPath, SupportsKeysAndGetItem
+from collections.abc import Callable, Iterable
+from typing import Any, AnyStr
+from typing_extensions import Literal, TypeAlias
+
+import pkg_resources
+from PyInstaller import HOMEPATH as HOMEPATH, isolated
+from PyInstaller.depend.imphookapi import PostGraphAPI
+from PyInstaller.utils.hooks import conda as conda_support
+from PyInstaller.utils.hooks.win32 import get_pywin32_module_file_attribute as get_pywin32_module_file_attribute
+
+_Environ: TypeAlias = SupportsKeysAndGetItem[str, str] | Iterable[tuple[str, str]] | os._Environ[str]
+
+PY_IGNORE_EXTENSIONS: set[str]
+hook_variables: dict[str, str]
+
+def exec_statement(statement: str) -> str | int: ...
+def exec_statement_rc(statement: str) -> str | int: ...
+def exec_script(script_filename: os.PathLike[AnyStr] | AnyOrLiteralStr, *args: str, env: _Environ | None = ...) -> str | int: ...
+def exec_script_rc(
+    script_filename: os.PathLike[AnyStr] | AnyOrLiteralStr, *args: str, env: _Environ | None = ...
+) -> str | int: ...
+def eval_statement(statement: str) -> Any | Literal[""]: ...
+def eval_script(
+    scriptfilename: os.PathLike[AnyStr] | AnyOrLiteralStr, *args: str, env: _Environ | None = ...
+) -> Any | Literal[""]: ...
+@isolated.decorate
```

Comment:
We usually omit decorators in stub files -- there are very limited exceptions, such as `@classmethod`, `@staticmethod`, and `@dataclass`, but outside of those, it's generally considered undefined behaviour in a stub.

```suggestion
```